README.md

```diff
@@ -1,2 +1,270 @@
+
+### /users/$userId$/medicationRequests/$medicationRequestId$
+
+Based on [FHIR MedicationRequest](https://hl7.org/fhir/R4B/medicationrequest.html), the following properties may be used, while additional properties are ignored by the Engage-HF system.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|medication|Reference(Medication) or CodeableConcept|-|TBD: `medicationId` as used in /medications/$medicationId$ or a CodeableConcept containing the same information as one of the medications listed in /medications/$medicationId$|
+|dosageInstruction|list of Dosage|-|-|
+|patient|string|e.g. "allergy", "intolerance"|`userId` as used in /users/$userId$ and related collections.|
+
+The `dosageInstruction` property may contain values containing the following properties:
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|text|optional string|-|Free text dosage instructions|
+|additionalInstruction|list of CodableConcept|information like "with meals", "may cause drowsiness", etc|Supplemental instruction or warnings to the patient|
+|patientInstruction|optional string|-|Patient or consumer oriented instructions|
+|timing|optional [Timing](https://hl7.org/fhir/R4B/datatypes.html#timing)|-|When medication should be administered|
+|doseAndRate|list of Element|-|Amount of medication administered|
+|doseAndRate>type|optional [DoseRateType](https://hl7.org/fhir/R4B/codesystem-dose-rate-type.html)|e.g. "calculated", "ordered"||
+|doseAndRate>dose|optional [Range](https://hl7.org/fhir/R4B/datatypes.html#Range) or [SimpleQuantity](https://hl7.org/fhir/R4B/datatypes.html#SimpleQuantity)|-|-|
+|doseAndRate>rate|optional [Ratio](https://build.fhir.org/datatypes.html#ratio) or [Range](https://hl7.org/fhir/R4B/datatypes.html#Range) or [SimpleQuantity](https://hl7.org/fhir/R4B/datatypes.html#SimpleQuantity)|-|-|
+|maxDosePerPeriod|optional [Ratio](https://build.fhir.org/datatypes.html#ratio)|-|Upper limit on medication per unit of time|
+|maxDosePerAdministration|optional [SimpleQuantity](https://build.fhir.org/datatypes.html#SimpleQuantity)|-|Upper limit on medication per administration|
+|maxDosePerLifetime|optional [SimpleQuantity](https://build.fhir.org/datatypes.html#SimpleQuantity)|-|Upper limit on medication per unit of time|
+
+TBD: Do we need to be able to specify rate vs dose? i.e. are there scenarios where a clinician would like to just specify "take this amount in this time" rather than "take one pill every evening"?
+
+TBD: Which timings, doses and rates does the system need to be able to work with? How does this differ between input and processing (e.g. an algorithm might only be interested in the total amount administered per day/week)?
```

Comment:
Would be great if we can document the output of the meeting here.

Reply:
Yes, I will add that the list has up to 3 items with timings being morning/midday/evening and some specific doses (0.5, 1 or 2 pills). Since the maximum/target daily dose is in grams, we might need some information about how much grams is one pill per medication.

Reply:
100%, this needs to be encoded in the MedicationRequest. Normally medication codes are unique (name of medication + dosage) so these are clearly defined as part of the medication code. If we don't use standardized codes we might need to encode this as part of the medication code.

Reply:
I added the medication (class) codes to the allergyIntolerance chapter, most of them were available on SNOMED CT, for the rest I wrote TBD for now.